src/sources/util/unix_stream.rs

```diff
@@ -133,9 +133,6 @@ pub fn build_unix_stream_source(
             );
         }
 
-        // Cleanup
-        drop(stream);
```

Comment:
Apparently `stream` didn't implement `Drop` so dropping it here had no real effect.